README.md

```diff
@@ -74,6 +74,8 @@ Zettlr is an [Electron](https://www.electronjs.org/)-based app, so to start deve
 1. A [NodeJS](https://nodejs.org/)-stack installed on your computer. Make sure it's at least Node 14 (`lts/fermium`). To test what version you have, run `node -v`.
 2. [Yarn](https://yarnpkg.com/en/) installed. Yarn is the required package manager for the project, as we do not commit `package-lock.json`-files and many commands require yarn. You can install this globally using `npm install -g yarn` or Homebrew, if you are on macOS.
 
+Note that [cURL](https://curl.se/download.html) library needs to be installed on your computer.
```

Comment:
```suggestion
3. [cURL](https://curl.se/download.html). This is required to fetch Pandoc in the background.
```

Reply:
Sounds good ! Should I do something to do the PR keeps going ?